extensions/sql-bindings/src/sql-bindings.d.ts

```diff
@@ -0,0 +1,97 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+
+declare module 'sql-bindings' {
+
```

Comment:
I would also suggest adding a const for the extension name like we do in other typings files - it makes it easier and safer to get the name when loading the extension API.